components/YoutubeVideoPlayer/index.tsx

```diff
@@ -189,23 +226,50 @@ const YoutubeVideoPlayer: FC<Props> = ({ videoId }) => {
   return <>
     <div className="flex items-center">
       { hasControl ? <>
-        <h3>You are host</h3>
+        <h3 className="p-4">You are host of party <strong>{partyId}</strong></h3>
         <button
           onClick={handleCopyLink}
           className="m-2 flex items-center justify-center px-5 py-3 border border-transparent text-base font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-700"
         >
-          Click to Copy Share Link
+          Copy Share Link
         </button>
       </> : <div>
-        <h3>{'Someone else is host.'}</h3>
+        <h3 className="p-4">Someone else is host of party <strong>{partyId}</strong></h3>
       </div> }
     </div>
-    <div id='youtube-video' style={PLAYER_STYLE} />
+    { /* https://stackoverflow.com/a/49887085/2696867 */ }
+    <style dangerouslySetInnerHTML={{__html: `
```

Comment:
would this work to not have to have the embedded style tags? https://github.com/cristianbote/goober